__init__.py

```diff
@@ -0,0 +1 @@
+# Shahaf look at me

```

Comment:
Please ensure coverage with UTs.

Reply:
This also implies the existence of a `Shahaf` at all times. Consider making a variable.

Reply:
> This also implies the existence of a `Shahaf` at all times. Consider making a variable.

he is here to stay

Reply:
🤔